images/capi/packer/ami/flatcar.json

```diff
@@ -1,9 +1,10 @@
 {
-  "ami_filter_name": "Flatcar*stable*",
+  "ami_filter_name": "Flatcar*beta*",
   "ami_filter_owners": "075585003325",
   "ansible_extra_vars": "ansible_python_interpreter=/opt/bin/python",
-  "build_name": "flatcar-stable",
+  "build_name": "flatcar-beta",
```

Comment:
this switch is to overcome the outdated stuff in stable?

Reply:
> this switch is to overcome the outdated stuff in stable?

Exactly.
Afterwards when the stable had containerd 1.3, we could set it back to flatcar-stable.